econpizza/config.py

```diff
@@ -0,0 +1,76 @@
+"""Configuration object"""
+
+import os
+import jax
+
+class EconPizzaConfig(dict):
+    def __init__(self, *args, **kwargs):
+        super(EconPizzaConfig, self).__init__(*args, **kwargs)
+        self._enable_persistent_cache = False
+        self._cache_folder_pizza = None
+        self._cache_folder_jax = None
+
+    @property
+    def enable_persistent_cache(self):
+        return self._enable_persistent_cache
+
+    @enable_persistent_cache.setter
+    def enable_persistent_cache(self, value):
+        self._enable_persistent_cache = value
+
+    @property
+    def cache_folder_jax(self):
+        return self._cache_folder_jax
+    
+    @cache_folder_jax.setter
+    def cache_folder_jax(self, value):
+        self._cache_folder_jax = value
+    
+    @property
+    def cache_folder_pizza(self):
+        return self._cache_folder_pizza
+    
+    @cache_folder_pizza.setter
+    def cache_folder_pizza(self, value):
+        self._cache_folder_pizza = value
+
+    def update(self, key, value):
+        if hasattr(self, key):
+            setattr(self, key, value)
+        else:
+            raise AttributeError(f"'EconPizzaConfig' object has no attribute '{key}'")
+
+    def __repr__(self):
+        properties = {
+            k.lstrip("_"): v for k, v in self.__dict__.items() if k.startswith("_")
+        }
+        return f"{properties}"
+
+    def __str__(self):
+        return self.__repr__()
+
+
+config = EconPizzaConfig()
+
+def _create_cache_dir(folder_name: str):
+    cwd = os.getcwd()
+    folder_path = os.path.join(cwd, folder_name)
+    os.makedirs(folder_path, exist_ok=True)
+
+    return folder_path
+
```

Comment:
This way the cache folders are created in the directory from where the script is called, so I think the Windows issue is resolved

Reply:
Yes, this should be good.